contracts/Registrar.sol

```diff
@@ -163,4 +163,24 @@ contract Registrar is OpsManaged {
 			_brandedToken,
 			_checkUuid);
 	}
+
+	/*
+     *  OpenSTValue
+     */
+    function revertUnstaking(
+        // address of OpenSTValue registry:
+        OpenSTValueInterface _registry,
+        // OpenSTValue function:
+        bytes32 _redemptionIntentHash
+        )
+        external
+        onlyOps
```

Comment:
@benjaminbollen please confirm if onlyOps is fine or it should be onlyAdminOrOps?
confirmRedemptionIntent is onlyOps others are onlyAdminOrOps.